### PR TITLE
Section 508 External blocks for Open ACR

### PR DESCRIPTION
**Pull request by HarikaMunipalle**

### CODE REVIEW

**Comment by gsa-bri** [comment]
Hi Harika,

Got your req for PR 747 review. Will you briefly explain what the external
blocks are?

Thanks,
Bri

**Comment by HarikaMunipalle** [comment]
Hi Brianna,

The external blocks code is to utilize the Section 508 header and footer to
Open ACR.

Thanks,
Harika

**Comment by gsa-bri** [comment]
Got it, thanks!

File: assets/js/external-508-blocks.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+    sidenav: false
+---
+    {% assign myUrlPrefix = '' %}
+{% if site.baseurl contains '/preview' %}
+{% assign myUrlPrefix = 'https://federalist-7a132a2e-6307-4cd0-9f82-e30e871d214a.sites.pages.cloud.gov' %}
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply assets/js/external-508-blocks.js:6]: @HarikaMunipalle, can you take a look at this part of the javascript to confirm approach? This appears to point just to this specific branch preview, not necessarily to other possible future branch previews.

**Review by drewnielson — APPROVED** [review]

**Pull request merged**